configs/webpack.config.common.js

```diff
@@ -110,4 +110,50 @@ const previewConfig = {
     ]
 };
 
-module.exports = [widgetConfig, previewConfig];
+const configurations = [widgetConfig, previewConfig];
```

Comment:
const configurations = {
  widgetConfig: widgetConfig,
  previewConfig: previewConfig
}

Reply:
nope.. is an array of configs, not an object